src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -238,12 +256,12 @@ uiModules.get('apps/management')
       : undefined;
 
     // this seems wrong, but it's the original logic... https://git.io/vHYFo
-    const notExpandable = (!expandable && this.canExpandIndices())
+    const notExpandable = (this.canExpandIndices() && !expandable)
       ? true
       : undefined;
 
     // Only event-time-based index patterns set an intervalName.
-    const intervalName = (nameIsPattern && nameInterval)
+    const intervalName = (this.canUseTimePattern() && nameIsPattern && nameInterval)
```

Comment:
Well, ng-if doesn't unset the model value when the inputs are removed from the view, so it's possible that the name interval is still set when the name isn't actually a time pattern